ONSdigital/response-operations-ui
# Fixing collection exercise page to deal with sample load error

# What and why?
At the moment, if the sample is in the process of being loaded, the `ingestDateTime` value is `None` which causes a 500 error since one of the tables is trying to subscript a NoneType. This adds a check to ensure that the sample upload date time is only being accessed once the sample is fully loaded.
# How to test?
Go into the database and change the `state_fk` column in the `sample_summary` table between `INIT`, `ACTIVE`, `FAILED` and `COMPLETE`, to check that none of the states cause an error.
# Trello
https://trello.com/c/i1CkfY6p/1739-500-on-rops-when-sample-is-being-loaded-causing-acceptance-tests-to-fail
